packages/ariakit/src/toolbar/__examples__/toolbar-with-menu/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { forwardRef } from "react";
+import { Button } from "ariakit/button";
+import { Menu, MenuButton, MenuItem, useMenuState } from "ariakit/menu";
+import {
+  Toolbar,
+  ToolbarItem,
+  ToolbarSeparator,
+  useToolbarState,
+} from "ariakit/toolbar";
+import { BiChevronDown } from "react-icons/bi";
+import {
+  ImBold,
+  ImItalic,
+  ImParagraphCenter,
+  ImParagraphLeft,
+  ImParagraphRight,
+  ImUnderline,
+} from "react-icons/im";
+import "./style.css";
+
+const MoreItems = forwardRef((props, ref) => {
+  const menu = useMenuState({ orientation: "horizontal" });
+  return (
+    <>
+      {/* @ts-ignore */}
```

Comment:
sorry about that, but i couldn't resolve it.

Reply:
What's the error message? Just let the CI throw it so we can find a solution.